### DISCUSSION TITLE
Generating all all_contig_annotations.json to run enclone after MiXCR

### DISCUSSION THREAD

**Discussion by JackYGhannam** [milaboratory/mixcr#1704] [General]
Hello! 

I’m currently running MixCR to analyze 10x 5’ VDJ + nanopore sequencing with a custom PCR-based enrichment protocol. After running MiXCR, I’d like to run 10x’s enclone pipeline for its clonotype and barcode filtering algorithm. 

In order to do so, I would need to generate a file identical to all_contig_annotations.json that is an output during cellranger, information which I believe MiXCR can output. 

Is there a way to configure MixCR to output such a file? I suspect this is a long shot, but it would be tremendously helpful. Thank you!

**Comment by mizraelson** [comment]
Hi, as for now MiXCR can't output an identical file to  cellranger. There is a multiple number of columns that you can use for the output that you can read about [here](https://mixcr.com/mixcr/reference/mixcr-export/#export-fields). MiXCR output can only be a compilation of these columns.

What command did you use for the data analysis?

**Reply by zztin** [reply]
Hi, Following up on this matter. I am also interested in running MiXCR for 10x 5'GEX or 3'GEX + Nanopore sequencing. What command would you suggest to use for this data type? 

I've started with one of the presets 
`mixcr analyze generic-ont-with-umi --species hsa --tag-pattern "^(UMI:N{12})(R1:*)" sample.fastq.gz  sample_result`
but I'm pretty sure that there are modifications to be made (for example the UMI sequences.) 

Since a preset is a required field. I'd like to hear from the team what you suggest to use!

Best, 
Li-Ting

References:

Documentation:
https://mixcr.com/mixcr/reference/overview-built-in-presets/

Github pages:
https://github.com/milaboratory/mixcr/blob/develop/src/main/resources/presets/protocols/10x.yaml
https://github.com/milaboratory/mixcr/blob/develop/src/main/resources/presets/protocols/nanopore.yaml

**Reply by mizraelson** [reply]
Hi, it is discussed here #1681
You would need a custom preset as mentioned there. Additionally, with ONT sequencing, you probably have adaptors before the CELL ID, so you would need to specify them in the tag pattern.
